<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BackupWindow</class>
 <widget class="QWidget" name="BackupWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>924</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/**
 * Blue: rgb(61, 149, 245);
 * Hover Blue: rgb(13, 122, 242);
 * Light Hover Blue: rgb(187, 225, 250);
 * Red: rgb(242, 22, 43);
 * Hover Red: rgb(203, 11, 29);
 * Green: rgb(82, 224, 102);
 * Yellow: rgb(253, 214, 52);
 * Hover Yellow:  rgb(253, 205, 36);
 * Hover Green: rgb(39, 216, 63);
 * Light Gray: rgb(200, 200, 200);
 * Gray : rgb(239, 239, 239);
 * White: rgb(248, 248, 248);
 */

#BackupWindow {
	background-color: rgb(239, 239, 239);
	border-radius: 10px;
}

#importButton {
	background-color: rgb(82, 224, 102);
}

#importButton:hover {
	background-color: rgb(39, 216, 63);
}

#manageButton {
	background-color: rgb(253, 214, 52);
}

#manageButton:hover {
	
	background-color: rgb(253, 205, 36);
}

#searchButton {
	border-radius: 15px;
}

.QTableWidget {
	background-color: rgb(248, 248, 248);
	padding: 10px;
	border: 2px solid rgb(200, 200, 200);
	border-radius: 5px;
}

QHeaderView::section {
	background-color:rgb(248, 248, 248);
	color: rgb(57, 59, 68);
	border: 2px solid rgb(200, 200, 200);
	padding-left: 5px;
	padding-right: 5px;
	border-radius: 5px;
}

QLabel {
	background-color: rgb(239, 239, 239);
}

QLineEdit {
	background-color: rgb(248, 248, 248);
	border-radius: 5px;
	border: 2px solid rgb(200, 200, 200);
}

QLineEdit:hover {
	border: 2px solid rgb(187, 225, 250);
}

QLineEdit:focus {
	border: 2px solid rgb(13, 122, 242);
}

QSpinBox {
	background-color: rgb(248, 248, 248);
	border-radius: 5px;
	border: 2px solid rgb(200, 200, 200);
}

QSpinBox:hover {
	border: 2px solid rgb(187, 225, 250);
}

QSpinBox:focus {
	border: 2px solid rgb(13, 122, 242);
}

QPlainTextEdit {
	background-color: rgb(248, 248, 248);
	border-radius: 5px;
	border: 2px solid rgb(200, 200, 200);
}

QPlainTextEdit:hover {
	border: 2px solid rgb(187, 225, 250);
}

QPlainTextEdit:focus {
	border: 2px solid rgb(13, 122, 242);
}

QPushButton {
	background-color: rgb(61, 149, 245);
	color: rgb(248, 248, 248);
	border-radius: 5px;
}

QPushButton:hover {
	background-color: rgb(13, 122, 242);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QFrame" name="backupFrame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLabel" name="backupLabel">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Montserrat Light</family>
          <pointsize>20</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">#backupLabel {
	color: rgb(13, 122, 242); 
}</string>
        </property>
        <property name="text">
         <string>Backup (Security Copy)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="controlsFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="exportButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Montserrat Medium</family>
          <pointsize>11</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::TabFocus</enum>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Export Backup</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/icons/downarrow.png</normaloff>:/icons/downarrow.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="importButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Montserrat Medium</family>
          <pointsize>11</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::TabFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">/** 
  * Green: rgb(82, 224, 102);
  * Darker green: rgb(39, 216, 63);
*/

#importBackupButton {
	background-color: rgb(82, 224, 102);
}

#importBackupButton:hover {
	background-color: rgb(39, 216, 63);
}</string>
        </property>
        <property name="text">
         <string>Import Backup</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/icons/uparrow.png</normaloff>:/icons/uparrow.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="managedLabel">
        <property name="font">
         <font>
          <family>Montserrat</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>No backup loaded</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="manageButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Montserrat Medium</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::TabFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Manage Backup</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/icons/supervisor.png</normaloff>:/icons/supervisor.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QFrame" name="backupSearchFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLineEdit" name="searchLineEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Montserrat</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="maxLength">
         <number>50</number>
        </property>
        <property name="placeholderText">
         <string>ID/Sender</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="searchButton">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Montserrat</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::TabFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/icons/search.png</normaloff>:/icons/search.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QTableWidget" name="mailTableWidget">
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>11</pointsize>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="columnCount">
      <number>4</number>
     </property>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Sender</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Subject</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Actions</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>exportButton</tabstop>
  <tabstop>importButton</tabstop>
  <tabstop>manageButton</tabstop>
  <tabstop>searchLineEdit</tabstop>
  <tabstop>searchButton</tabstop>
  <tabstop>mailTableWidget</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
